### PR TITLE
New responsive navigation

### PR DESCRIPTION
**Pull request by codecalm**
The entire top navigation has been completely rebuilt

https://deploy-preview-584--tabler.netlify.com/navigation.html

Screenshot 2020-04-05 at 15 22 30

### CODE REVIEW

**Comment by codecalm** [comment]
Screenshot 2020-04-05 at 16 03 13

File: scss/_variables.scss [hunk 1]
```diff
@@ -363,29 +365,61 @@ $modal-lg: 720px !default;
...
-$navbar-vertical-narrow-width: 4rem !default;
-$navbar-vertical-dark-bg: $dark !default;
-$navbar-vertical-dark-color: $white !default;
-$navbar-vertical-padding: 1rem !default;
-$navbar-vertical-active-accent-color: $primary !default;
-
-$navbar-nav-link-padding-x: .75rem !default;
+$navbar-toggler-padding-x: 0 !default;
+$navbar-toggler-padding-y: 0 !default;
+$navbar-light-toggler-icon-bg: url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" class="icon icon-tabler icon-tabler-align-justified" width="24" height="24" viewBox="0 0 24 24" stroke-width="2" stroke="' + $body-color + '" fill="none" stroke-linecap="round" stroke-linejoin="round"><path stroke="none" d="M0 0h24v24H0z"/><line x1="4" y1="6" x2="20" y2="6" /><line x1="4" y1="12" x2="20" y2="12" /><line x1="4" y1="18" x2="20" y2="18" /></svg>') !default;
```
Comment (WinterSilence) [inline scss/_variables.scss:396]: 24px != 1rem and it's not encoded string

File: scss/_variables.scss [hunk 2]
```diff
@@ -169,11 +169,9 @@ $spacers: (
   1: $spacer * .25,
   2: $spacer * .5,
   3: $spacer,
-  4: $spacer * 1.25,
-  5: $spacer * 2,
-  6: $spacer * 3,
-  7: $spacer * 6,
-  8: $spacer * 9,
+  4: $spacer * 1.5,
```
Comment (WinterSilence) [inline scss/_variables.scss:172]: why not use fixed step, for example `.25`: `1: $spacer * .5, 2: $spacer * .75, 3: $spacer * 1, 4: $spacer * 1.25, 5: $spacer * 1.5, 6: $spacer * 1.75,` ?

File: pages/icons.html [hunk 3]
```diff
@@ -4,11 +4,10 @@
 ---
 
 <div class="row">
-   <div class="col-12">
-      {% include cards/icons.html title="Tabler icons" group="tabler" %}
-   </div>
-
 	{% if jekyll.environment == 'production' %}
+	<div class="col-12">
```
Comment (WinterSilence) [inline pages/icons.html:8]: why only in production?

File: scss/_variables.scss (hunk 1 above)
Comment (WinterSilence) [reply scss/_variables.scss:396]: @codecalm
~~~
$form-icon-size: calculatePx($form-check-input-width) - $form-check-input-border * 2 !default;
$form-check-input-checked-bg-image: url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' width='{$form-icon-size}' height='{$form-icon-size}' viewBox='0 0 {$form-icon-size} {$form-icon-size}'><path fill='none' stroke='#{$form-check-input-checked-color}' stroke-linecap='round' stroke-linejoin='round' stroke-width='3' d='M4 8.5L6.5 11l6-6' transform='scale(.8)' transform-origin='50% 50%'/></svg>") !default;
$form-check-input-checked-bg-image-dark: url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' width='{$form-icon-size}' height='{$form-icon-size}' viewBox='0 0 {$form-icon-size} {$form-icon-size}'><path fill='none' stroke='#{$dark}' stroke-linecap='round' stroke-linejoin='round' stroke-width='3' d='M4 8.5L6.5 11l6-6' transform='scale(.8)' transform-origin='50% 50%'/></svg>") !default;
$form-check-radio-checked-bg-image: url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' width='{$form-icon-size}' height='{$form-icon-size}' viewBox='0 0 {$form-icon-size} {$form-icon-size}'><circle r='3' fill='#{$form-check-input-checked-color}' cx='8' cy='8' /></svg>") !default;
~~~

**Comment by frknfndk** [comment]
Hello @codecalm ,

When we can get this navigation fixes in dev with a build? We are so excited to use it :)

**Comment by codecalm** [comment]
sooon :)

**Comment by frknfndk** [comment]
> sooon :)

thank you :+1:

File: scss/ui/_buttons.scss [hunk 4]
```diff
@@ -1,5 +1,5 @@
 .btn {
-  font-weight: 600;
+  font-weight: 500;
```
Comment (WinterSilence) [inline scss/ui/_buttons.scss:2]: use constants as bold, normal - it's more useful - not all fonts have size as 800,400.

**Comment by codecalm** [comment]
merged! 🎉👨‍🎨

**Pull request merged**